### DISCUSSION TITLE
Make `toBuilder` and `formBuiler` APIs more unified

### DISCUSSION THREAD

**Discussion by 4eUeP** [ZHaskell/z-data#7] [General]
For now, if we want to build a `Builder` from `bytes`, `int` ... we can find these functions under `Z.Data.Builder`, but the `CBytes`'s `toBuilder` is under `Z.Data.CBytes`.

What about putting these builder functions in one place? @winterland1989

**Comment by winterland1989** [comment]
CBytes is a more independent module, so I didn’t want to make a reversed dependency. And if we move them to Builder then we have to explain why it may write invalid UTF8 bytes, which is somewhat duplicated.

**Reply by 4eUeP** [reply]
Let's move this issue to the discussion and wait for more user feedback.
